docs/community_snippets.md

```diff
@@ -244,15 +244,15 @@ Quickly create a Pester Test for existence of a mandatory parameter by @SQLDBAWi
 #### Snippet
 
 ```json
-    "Pester for Mandatory Pester": {
-        "prefix": "mandatoryParamPester",
-        "body": [
-            "It \"${1:FunctionName} Should have a mandatory parameter ${2:ParameterName}\" {",
-            "   (Get-Command ${1:FunctionName}).Parameters['${2:ParameterName}'].Attributes.Mandatory | Should -BeTrue",
-            "}"
-        ],
-        "description": "Pester Test for Parameter"
-    }
+"Pester for Mandatory Pester": {
```

Comment:
Should this be `Pester for Mandatory Parameter`?

Reply:
I guess so, based on the prefix and h3.
But I didn't change any content to the snips, as they are not mine.
(https://github.com/PowerShell/vscode-powershell/pull/1297)

Let me know if it is desired to fix that with this PR. I don't mind

Reply:
Yeah, that was just a typo from the original PR.  Would you mind fixing it?  Thanks.